### PR TITLE
cloudflare alerts

### PR DESCRIPTION
**Pull request by mahirchavda**

### CODE REVIEW

File: cyences_app_for_splunk/bin/cs_product_list.py [hunk 1]
```diff
@@ -451,6 +451,25 @@ def build_app_dependency_search(app_list):
...
+                "link": "https://splunkbase.splunk.com/app/4501"
+            }
+        ],
+        "macro_configurations": [
+            {
+                "macro_name": "cs_cloudflare",
+                "label": "Cloudflare Data",
+                "search_by": "sourcetype",
+                "search_values": "cloudflare:json",
+                "earliest_time": "-4h@h",
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/bin/cs_product_list.py:468]: Is 4 hours enough to make sure some data present in this duration?

File: cyences_app_for_splunk/default/savedsearches.conf [hunk 2]
```diff
@@ -3778,6 +3778,75 @@ action.cyences_notable_event_action.attacker_search = | stats count by ip_client
 action.cyences_notable_event_action.attacker_drilldown =  `cs_f5_bigip_asm` ip_client="*" dest_ip="*" severity IN ("critical","high","medium") enforcement_action!=block attack_type!="JSON Parser Attack" ip_client=$row.ip_client$
 action.cyences_send_email_action = 1
 action.cyences_notable_event_action.products = F5 BIGIP
+action.cyences_notable_event_action.teams =
```
Comment (VatsalJagani) [inline cyences_app_for_splunk/default/savedsearches.conf:3781]: This is missing value SOC

**Review by VatsalJagani — CHANGES_REQUESTED** [review]
Need to create separate PR for Cloudflare data collection in the Docs section.

File: cyences_app_for_splunk/bin/cs_product_list.py (hunk 1 above)
Comment (mahirchavda) [reply cyences_app_for_splunk/bin/cs_product_list.py:468]: Yes

**Review by hardikhdholariya — APPROVED** [review]

**Comment by mahirchavda** [comment]
Will Create new PR for doc

**Review by VatsalJagani — APPROVED** [review]

**Pull request merged**